test/api_integration/apis/saved_objects/index.js

```diff
@@ -19,17 +19,17 @@
 
 export default function ({ loadTestFile }) {
   describe('saved_objects', () => {
-    loadTestFile(require.resolve('./bulk_create'));
-    loadTestFile(require.resolve('./bulk_get'));
-    loadTestFile(require.resolve('./create'));
-    loadTestFile(require.resolve('./delete'));
-    loadTestFile(require.resolve('./export'));
+    // loadTestFile(require.resolve('./bulk_create'));
+    // loadTestFile(require.resolve('./bulk_get'));
+    // loadTestFile(require.resolve('./create'));
+    // loadTestFile(require.resolve('./delete'));
+    // loadTestFile(require.resolve('./export'));
     loadTestFile(require.resolve('./find'));
-    loadTestFile(require.resolve('./get'));
-    loadTestFile(require.resolve('./import'));
-    loadTestFile(require.resolve('./resolve_import_errors'));
-    loadTestFile(require.resolve('./update'));
-    loadTestFile(require.resolve('./bulk_update'));
-    loadTestFile(require.resolve('./migrations'));
+    // loadTestFile(require.resolve('./get'));
+    // loadTestFile(require.resolve('./import'));
+    // loadTestFile(require.resolve('./resolve_import_errors'));
+    // loadTestFile(require.resolve('./update'));
+    // loadTestFile(require.resolve('./bulk_update'));
+    // loadTestFile(require.resolve('./migrations'));
```

Comment:
more tests that should be put back

Reply:
I would like to vulgar with myself right now